contracts/GenArt721CoreV3.sol

```diff
@@ -884,6 +894,18 @@ contract GenArt721CoreV3 is
         emit ProjectUpdated(_projectId, FIELD_PROJECT_BASE_URI);
```

Comment:
Right now Hasura tracks this data as `token_base_url` IIRC. This is used within the token api for some logic to how to construct some pieces of data (external_url). Should we sync contract base uri as default as well? Or will this happen automatically with inheritance here as structured?

Reply:
Very much believe we should also sync contract base URI as well, and plan to update the V3 Integration design doc with a `contract.baseURI` field in our subgraph, and we will want to sync to our DB as well 😄